discoverer/src/main/java/eu/seaclouds/platform/discoverer/service/DiscovererApplication.java

```diff
@@ -0,0 +1,30 @@
+package eu.seaclouds.platform.discoverer.service;
+
+import com.mongodb.MongoClient;
+import eu.seaclouds.platform.discoverer.api.*;
+import eu.seaclouds.platform.discoverer.core.Discoverer;
+import eu.seaclouds.platform.discoverer.core.DiscovererConfiguration;
+import io.dropwizard.Application;
+import io.dropwizard.setup.Environment;
+
+public class DiscovererApplication extends Application<DiscovererConfiguration> {
+
+    public static void main(String[] args) throws Exception {
+        new DiscovererApplication().run(args);
+    }
+
+    @Override
+    public void run(DiscovererConfiguration configuration, Environment environment) {
+
+        MongoClient mongoClient = new MongoClient(configuration.getDatabaseURL(), configuration.getDatabasePort());
+
+        Discoverer discoverer = new Discoverer(mongoClient, configuration.getActiveCrawlers());
```

Comment:
why the `Discoverer` object needs to be exposed here? Wouldn't be enough to pass the `mongoClient` to the `*API` ?